_sass/_404.scss

```diff
@@ -0,0 +1,257 @@
+.cls-1 {
```

Comment:
There seem to be a lot of things in here thatt i dont think we use

Reply:
again i have no intention of messing with the 404 page, it Just Works